### ISSUE TITLE
Error: "hdb" could not be required. Please make sure it is installed.

### ISSUE THREAD

**Issue by gregorwolf** [sapops/hana-cli#1]
Hi @ThePlenkov,

with the commit https://github.com/gregorwolf/bookshop-demo/commit/f282cb6f279578a06edca307193b4006d3baa600 I've added hana2cds to my sample project. 

I've stored the credentials with this commands:

```
cds bind --to bookshop-demo-db:bookshop-demo-db-key
cds env get requires.db.credentials --resolve-bindings --profile hybrid | json2fs --to .cds/requires/db/credentials
```

But when I run:

```
CDS_ENV=hybrid hana2cds -s PUBLIC -f M_OBJECT_LOCKS                 
```

I get this error:

```
/Users/gwolf/.nvm/versions/node/v16.19.0/lib/node_modules/hana2cds/node_modules/@sap/cds/libx/_runtime/hana/driver.js:196
      throw new Error(`"${name}" could not be required. Please make sure it is installed.`)
            ^

Error: "hdb" could not be required. Please make sure it is installed.
    at _getHanaDriver (/Users/gwolf/.nvm/versions/node/v16.19.0/lib/node_modules/hana2cds/node_modules/@sap/cds/libx/_runtime/hana/driver.js:196:13) 
```

Looking forward for your support.

CU
Gregor

**Comment by ThePlenkov** [comment]
on it

**Comment by ThePlenkov** [comment]
oh ok - I see. Could you please also try to add `@sap/hana-client`?

**Comment by ThePlenkov** [comment]
I probably need to add it as a peer dependency. This is how cds works - when it tries to access hana - either @sap/hana-client or hdb are needed.

**Comment by ThePlenkov** [comment]
Fixed in release 0.0.18. Please update the dependency. Let me know please if it helped
